admin/tests/acceptance/authenticated/certifications/certification/informations_test.js

```diff
@@ -299,36 +299,36 @@ module('Acceptance | Route | routes/authenticated/certifications/certification |
     module('when candidate results edit button is clicked', function () {
       test('it disables candidate informations edit button', async function (assert) {
         // when
-        await visit(`/certifications/${certification.id}`);
+        const screen = await visitScreen(`/certifications/${certification.id}`);
         await clickByName('Modifier les résultats du candidat');
 
         // then
-        assert.dom('[aria-label="Modifier les informations du candidat"]').isDisabled();
+        assert.dom(screen.getByLabelText('Modifier les informations du candidat')).isDisabled();
       });
     });
 
     module('when candidate results form cancel button is clicked', function () {
       test('it re-enables candidate informations edit button', async function (assert) {
         // when
-        await visit(`/certifications/${certification.id}`);
+        const screen = await visitScreen(`/certifications/${certification.id}`);
         await clickByName('Modifier les résultats du candidat');
         await clickByName('Annuler la modification des résultats du candidat');
 
         // then
-        assert.dom('[aria-label="Modifier les informations du candidat"]').exists().isEnabled();
+        assert.dom(screen.getByLabelText('Modifier les informations du candidat')).exists().isEnabled();
       });
     });
 
     module('when candidate results form is submitted', function () {
       test('it also re-enables candidate informations edit button', async function (assert) {
         // when
-        await visit(`/certifications/${certification.id}`);
+        const screen = await visitScreen(`/certifications/${certification.id}`);
         await clickByName('Modifier les résultats du candidat');
         await clickByName('Enregistrer les résultats du candidat');
         await clickByName('Confirmer');
 
         // then
-        assert.dom('[aria-label="Modifier les informations du candidat"]').exists().isEnabled();
+        assert.dom(screen.getByLabelText('Modifier les informations du candidat')).exists().isEnabled();
```

Comment:
**remarque**
Tiens je connaissais pas le `isEnabled`, apparament il servirait à tester si un element (input, textarea, select, bouton, options, fieldset...)  est actif ou pas ... 
https://github.com/simplabs/qunit-dom/blob/bcb67e0edaf470c887f6bded3c39c32d46f3ed3a/lib/assertions/is-disabled.ts

Reply:
Oui il y a plein d'assertions très pratiques listées ici : https://github.com/simplabs/qunit-dom/blob/master/API.md !